src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneModels/DataLakeStoreFileSystemClient.cs

```diff
@@ -68,6 +77,22 @@ public DataLakeStoreFileSystemClient(IAzureContext context)
             ServicePointManager.UseNagleAlgorithm = false;
         }
 
+        public DataLakeStoreFileSystemClient(IAzureContext context, DataLakeStoreFileSystemCmdletBase cmdlet) : this(context)
+        {
+            
```

Comment:
@rahuldutta90 nit: please remove this blank line

Reply:
@cormacpayne Removed it.